.gitignore

```diff
@@ -22,3 +22,4 @@ site/openstates
 +*.sublime*
 openstates.sqlite3
 dump/
+venv
```

Comment:
I'll pull this in because it's harmless, but you should check out `virtualenvwrapper` -- it's really great! Helps with virtual env management